flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/catalog/GenericInMemoryCatalogTest.java

```diff
@@ -562,6 +568,158 @@ public void testRenameView() throws Exception {
 		catalog.dropTable(viewPath2, false);
 	}
 
+	// ------ functions ------
+
+	@Test
+	public void testCreateFunction() throws Exception {
+		catalog.createDatabase(db1, createDb(), false);
+
+		assertFalse(catalog.functionExists(path1));
+
+		catalog.createFunction(path1, createFunction(), false);
+
+		assertTrue(catalog.functionExists(path1));
+
+		catalog.dropFunction(path1, false);
```

Comment:
we don't need these two drop commands, as they are already taken care in close()

Reply:
I think having it here is okay so that it doesn't pass the side-effect to other test cases.

Reply:
close() will be called after each unit test finishes to remove side-effects between tests. Leaving dropFunction() within unit tests, for example here, may mislead people that we are intentionally testing it, though it's only used for cleanup purpose.